include/agl/detail/aglGPUMemBlockMgr.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+
+#include <agl/aglGPUCommon.hpp>
+#include <container/seadPtrArray.h>
+#include <heap/seadDisposer.h>
+#include <hostio/seadHostIONode.h>
+#include <math/seadMathCalcCommon.h>
+#include <nvn/nvn_types.h>
+#include <prim/seadBitFlag.h>
+#include <prim/seadTypedBitFlag.h>
+#include <thread/seadCriticalSection.h>
+
+namespace agl::detail {
+
+typedef sead::BitFlag32 MemoryPoolDriverBitFlag;
+
+const s32 VALID_POOL_TYPE_VALUE = -1;
```

Comment:
```suggestion
constexpr s32 VALID_POOL_TYPE_VALUE = -1;
```
(same for the two constants below)